conformance/mesh/manifests.yaml

```diff
@@ -0,0 +1,130 @@
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: gateway-conformance-mesh
+  labels:
+    gateway-conformance: infra
+---
+
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: echo-v1
+  namespace: gateway-conformance-mesh
+  labels:
+    app: echo
+spec:
+  selector:
+    matchLabels:
+      app: echo
+      version: v1
+  template:
+    metadata:
+      labels:
+        app: echo
+        version: v1
+    spec:
+      containers:
+      - name: echo
+        image: gcr.io/istio-testing/app:latest
```

Comment:
I'd rather have this image owned by k8s/gateway-api infra so we can bundle versions of it with each release of our conformance tests. Can you create an issue to track following up on that? Shorter term can we at least use a pinned version of this image so a change in this image doesn't have an immediate impact on our conformance tests?

Reply:
https://github.com/kubernetes-sigs/gateway-api/issues/1895 and moved to a fixed image tag